src/boot/browser-check.js

```diff
@@ -9,6 +9,8 @@
  * @return {boolean}
  */
 export function isBrowserSupported() {
+  // Checks that return a truthy value if they succeed and throw or return
```

Comment:
Though every test here is well-commented and clear as to what it's up to, I don't entirely get a sense of "here's the method for how we selected these particular browser checks." I'm not too fussed about it right now, as it is certainly organized well.